docker/packager/packager

```diff
@@ -258,6 +258,10 @@ def parse_env_variables(
         if clang_tidy:
             # 15G is not enough for tidy build
             cache_maxsize = "25G"
+
+            # `CTCACHE_DIR` has the same purpose as the `CCACHE_DIR` above.
+            # It's there to have the clang-tidy cache embedded into our standard `CCACHE_DIR`
+            result.append("CTCACHE_DIR=/ccache/clang-tidy")
```

Comment:
It "feels" a bit weird to embed one cache directory into the other. Sounds safer to have both side-by-side. Also, the name "clang-tidy-cache/" would be more fitting.

Reply:
No problem to rename, but the overhead to have another cache is too much. TOO much

Reply:
I am just afraid that ccache will mess up its own cached artefacts with the clang-tidy-cache ones.

Reply:
Looks pretty good so far

https://github.com/ClickHouse/ClickHouse/actions/runs/3384444671/jobs/5621834614#step:7:575 +  
https://github.com/ClickHouse/ClickHouse/actions/runs/3384444671/jobs/5621834614#step:7:13233